api/client/webclient/webclient.go

```diff
@@ -396,8 +396,8 @@ func extractHostPort(addr string) (string, error) {
 	}
 }
 
-// extractHost takes addresses like "tcp://host:port/path" and returns "host".
-func extractHost(addr string) (ra string, err error) {
+// ExtractHost takes addresses like "tcp://host:port/path" and returns "host".
+func ExtractHost(addr string) (ra string, err error) {
```

Comment:
If we're going to export this, it seems like something that belongs in `lib/utils`.
We've got a variety of functions in there for parsing host/port/addr etc.